.sun-ci.yml

```diff
@@ -0,0 +1,35 @@
+stages: 
+  - build
+  - test
+
+jobs: 
+  build_and_lint:
+    stage: build
+    image: node:16-alpine
+    script: 
+      - yarn install --frozen-lockfile
+      - yarn eslint .
+  
+  test_and_coverage: 
+    stage: test
+    image: node:16-alpine
+    cache:
+      - key:
+          files:
+            - server/yarn.lock
+        paths:
+          - server/node_modules
+    script:
+      - cd server
+      - yarn install
```

Comment:
setting up the environment should be done on the build stage. Also, Please include frontend as well, you can separete the stages for testing FE and BE.